flow/middleware/logging.go

```diff
@@ -4,58 +4,62 @@ import (
 	"context"
 	"fmt"
 	"log/slog"
+	"time"
 
 	"google.golang.org/genproto/googleapis/api/annotations"
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/reflect/protoreflect"
 	"google.golang.org/protobuf/reflect/protoregistry"
 
+	"github.com/PeerDB-io/peerdb/flow/generated/protos"
 	"github.com/PeerDB-io/peerdb/flow/internal"
 )
 
-const grpcFullServiceName = "peerdb_route.FlowService"
+var GrpcFullServiceName = protos.FlowService_ServiceDesc.ServiceName
```

Comment:
@jgao54 I made a small change to refer to the string from the generated grpc code